### PR TITLE
A shit ton of renames

### PR DESCRIPTION
**Pull request by turian**

### CODE REVIEW

File: examples/examples.ipynb [hunk 1]
```diff
@@ -671,7 +671,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "drum1 = TorchDrum(\n",
+    "drum1 = Voice(\n",
```
Comment (jorshi) [inline examples/examples.ipynb:674]: lets rename drum1 to voice1 too?

File: tests/test_module.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+"""
+
+import pytest
+import torch.tensor as T
+
+import torchsynth.module as synthmodule
+from torchsynth.parameter import ModuleParameter, ModuleParameterRange
+
+
+class TestTorchSynthModule:
```
Comment (jorshi) [inline tests/test_module.py:12]: this should be renamed to TestSynthModule

File: tests/test_synth.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+import torch.tensor as T
+
+import torchsynth.globals
+import torchsynth.module as synthmodule
+import torchsynth.parameter
+import torchsynth.synth
+from torchsynth.parameter import ModuleParameter
+
+
+class TestTorchSynth:
```
Comment (jorshi) [inline tests/test_synth.py:17]: this should be renamed to TestAbstractSynth

**Review by jorshi — COMMENTED** [review]
This brings me great joy

File: examples/examples.ipynb (hunk 1 above)
Comment (jorshi) [reply examples/examples.ipynb:674]: drum2 below as well

Comment (turian) [reply examples/examples.ipynb:674]: done

File: tests/test_synth.py (hunk 3 above)
Comment (turian) [reply tests/test_synth.py]: done

**Pull request merged**